File: Try_GameEngine5/src/com/example/try_gameengine/framework/ITouchable.java
package com.example.try_gameengine.framework;

import android.view.MotionEvent;

public interface ITouchable extends ISystemTouchDelegate{
	public boolean onTouchBegan(MotionEvent event);
	public void onTouchMoved(MotionEvent event);
	public void onTouchEnded(MotionEvent event);
	public void onTouchCancelled(MotionEvent event);
}

File: Try_GameEngine5/src/com/example/try_gameengine/framework/ITouchable.kt
package com.example.try_gameengine.framework

import android.view.MotionEvent

interface ITouchable : ISystemTouchDelegate {
    fun onTouchBegan(event: MotionEvent?): Boolean
    fun onTouchMoved(event: MotionEvent?)
    fun onTouchEnded(event: MotionEvent?)
    fun onTouchCancelled(event: MotionEvent?)
}
